data/json/items/classes/magazine.json

```diff
@@ -5,6 +5,7 @@
     "name": { "str": "ammo belt" },
     "description": "An ammo belt consisting of metal linkages which disintegrate upon firing.",
     "volume": "0 ml",
```

Comment:
I would also update the volume value lest a singulartiy forms. Might also make sense to check zero volume since it can also trigger an error in `item::charges_per_volume`.

Reply:
In fact all volumes are forced to be at least 1ml already.  But you're right, it makes sense to report an error in similar circumstances.  Will do.